src/Admin/HostedServices/AzureQueueMailHostedService.cs

```diff
@@ -56,6 +59,10 @@ public async Task StopAsync(CancellationToken cancellationToken)
         private async Task ExecuteAsync(CancellationToken cancellationToken)
         {
             _mailQueueClient = new QueueClient(_globalSettings.Mail.ConnectionString, "mail");
+            var storageAccount = CloudStorageAccount.Parse(_globalSettings.Mail.ConnectionString);
+            _blobClient = storageAccount.CreateCloudBlobClient();
```

Comment:
Any advantage to having these available in the class scope versus local scope?

Reply:
Yeah, this is a hosted service and this method is the context that is split off as a new task. If there's an issue with the clients, we want to recreate them.  The task is long-lives so we're not recreating these unless we need to.

Reply:
I was mostly wondering why we store them as private variables in the class versus the scope since they only seems to be used inside the function.

Reply:
oh -- I see no reason, I was following the pattern of [EventsProcessor](https://github.com/bitwarden/server/blob/master/src/EventsProcessor/AzureQueueHostedService.cs#L25)